rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/facade/RpgParserFacade.kt

```diff
@@ -302,6 +302,7 @@ class RpgParserFacade {
         val parserResult: RpgParserResult
         val errors = LinkedList<Error>()
         val copyBlocks: CopyBlocks? = if (MainExecutionContext.getConfiguration().options.mustCreateCopyBlocks()) CopyBlocks() else null
+        // val includedCopySet = mutableSetOf<CopyId>()
```

Comment:
When can comments be left and when not?

Reply:
There are no rules, in that case I was not too sure about the feature I was implementing